app/assets/fonts/fontello/config.json

```diff
@@ -0,0 +1,94 @@
+{
+	"name": "placecal-icons",
+	"css_prefix_text": "icon-",
+	"css_use_suffix": false,
+	"hinting": true,
+	"units_per_em": 1000,
+	"ascent": 850,
+	"glyphs": [
+		{
+			"uid": "478c51ff6bd296826f1598a9de23e3a4",
+			"css": "time",
+			"code": 59396,
+			"src": "custom_icons",
+			"selected": true,
+			"svg": {
+				"path": "M555 520L501 574 490 562 429 502 377 449 449 377 501 429 609 322 681 394 555 520ZM501 168C317 168 168 318 168 502 168 686 317 835 501 835 685 835 834 686 834 502 834 318 685 168 501 168L501 168Z",
+				"width": 1000
+			},
+			"search": ["time"]
+		},
+		{
+			"uid": "cc6e45fb6f8f9329a4e7bd6d5ad53456",
+			"css": "repeats",
+			"code": 59395,
+			"src": "custom_icons",
+			"selected": true,
+			"svg": {
+				"path": "M1000 500L789 289 577 500 744 500C744 635 635 744 500 744L500 839C686 839 839 686 839 500L1000 500ZM500 256L500 161C314 161 161 314 161 500L0 500 211 711 423 500 256 500C256 365 365 256 500 256",
+				"width": 1000
+			},
+			"search": ["repeats"]
+		},
+		{
+			"uid": "dbfd56c8262ee93e49d548a82d461984",
+			"css": "place",
+			"code": 59394,
+			"src": "custom_icons",
+			"selected": true,
+			"svg": {
+				"path": "M501 538C436 538 384 485 384 421 384 356 436 304 501 304 566 304 618 356 618 421 618 485 566 538 501 538M501 129C340 129 210 260 210 421 210 676 501 874 501 874 501 874 793 676 793 421 793 260 662 129 501 129",
+				"width": 1000
+			},
+			"search": ["place"]
+		},
+		{
+			"uid": "fa9cb0d08cfa86be316ec6c596615c1f",
+			"css": "duration",
+			"code": 59393,
+			"src": "custom_icons",
+			"selected": true,
+			"svg": {
+				"path": "M293 268C342 290 414 314 500 314 586 314 658 290 707 268 689 371 603 449 500 449 397 449 311 371 293 268M592 500C699 460 775 355 775 230 775 170 698 133 601 118 585 116 568 114 551 113 534 112 517 111 500 111 483 111 466 112 449 113 432 114 415 116 399 118 302 133 225 170 225 230 225 355 301 460 408 500 301 540 225 645 225 770 225 929 775 929 775 770 775 645 699 540 592 500",
+				"width": 1000
+			},
+			"search": ["duration"]
+		},
+		{
+			"uid": "b912867e106c70b6407fdd8461e25b35",
+			"css": "date",
+			"code": 59392,
+			"src": "custom_icons",
+			"selected": true,
+			"svg": {
+				"path": "M771 711C771 744 744 771 711 771L289 771C256 771 229 744 229 711L229 470C229 437 256 410 289 410L711 410C744 410 771 437 771 470L771 711ZM741 229L741 199C741 166 714 139 681 139L651 139C617 139 590 166 590 199L590 229 410 229 410 199C410 166 383 139 350 139L319 139C286 139 259 166 259 199L259 229C193 229 139 283 139 349L139 741C139 807 193 861 259 861L741 861C807 861 861 807 861 741L861 349C861 283 807 229 741 229L741 229Z",
+				"width": 1000
+			},
+			"search": ["date"]
+		},
+		{
+			"uid": "34b24c0acb46f1e7ad858f4617a5e251",
+			"css": "online-alt",
```

Comment:
Are we using this one?

Reply:
tl;dr i think the non-alt one works better on a white background and the other one works better on a coloured background so itll be better for the admin interface